commands/addons/open.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+let cli = require('heroku-cli-util');
+let co  = require('co');
+let opn = require('opn');
+let fs  = require('fs');
+
+function open (url) {
+  cli.log(`Opening ${cli.color.cyan(url)}...`);
+  return new Promise(function (fulfill, reject) {
+    opn(url, {wait: false}, function (err) {
+      if (!err) return fulfill();
+      cli.error('Cannot open page.');
+      cli.error(err);
+      reject(new Error(`Open ${cli.color.cyan(url)} in your browser.`));
+    });
+  });
+}
+
+const ssoPath = '/tmp/heroku-sso.html';
+
+function writeTemplate (ctx, sso, path) {
+  return new Promise(function (fulfill, reject) {
+    let html = `<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>Heroku Add-ons SSO</title>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
+  </head>
+
+  <body>
+    <h3>Opening ${ctx.args.addon} on ${ctx.app}...</h3>
```

Comment:
(not that it matters so much since you'll see it for a split second, and only Herokai will se it)